index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <main id="main">
+        <h1 id="title"> Dr. Norman Borlaug </h1>
+        <p> The man who saved a billion lives </p>
+        <div id="img-div">
```

Comment:
1. I thing you should use section here instead of div.

Reply:
- Kindly, always includes an explanation, a "why", for each requested change. For example:
**Kindly, it would be better if you use a `section` tag for the element instead of a `div`, that would improve the HTML structure and make the page more accessible on search engines 👌**
- [Optional] I think using a couple of emojis and/or gifs in the review would be better. They make the code review more human-friendly :alien::robot:.

Reply:
Kindly, it would be better if you use a section tag for the element instead of a div, that would improve the HTML structure and make the page more accessible on search engines 👌

Actually, i don't know how to add gifs and emojis; i find it nowhere.

Reply:
i installing an extension for that purpose.